README.md

```diff
@@ -1,131 +1,236 @@
+<h2 id="table-of-contents"> 📑 Table of Contents</h2>
 
-The scope for `ae` will be for a cluster of Aurae nodes and will likely have identical functionality to `aer` in many places - but not all.
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li><a href="#about-the-project"> ► About The Project</a></li>
+    <li><a href="#quickstart"> ► Quickstart</a></li>
+    <li><a href="#usage"> ► Usage</a></li>
+    <li><a href="#philosophy"> ► Philosophy</a></li>
+    <li><a href="#contribute"> ► Contribute</a></li>
+  </ol>
+</details>
 
-The `ae` command line tool will be for working with a group of Aurae nodes, and will likely include more pragmatic functionality that is meaningful to an enterprise operator.
+&nbsp;
 
-### Who is ae for?
+<!-- ABOUT THE PROJECT -->
+## About The Project
 
-The `ae` tool should feel familiar to any cloud operator.
+`ae` is a UNIX inspired CLI client for **[Aurae](https://github.com/aurae-runtime/aurae)**, written in Go. However, in order to understand what `ae` should and can do, we must first understand `aer`.
 
--   Tasks such as "rolling out nginx to prod" should be possible with `ae`.
--   Tasks such as "counting the number of threads of a process" should be possible with `ae`.
--   Tasks such as "changing the current database connection count for a service" should be possible with `ae`.
--   Tasks such as "swinging traffic from one service to another" should be possible with `ae`.
-
-These are more "practical" or "outcome driven" tasks, and will likely require additional functionality to the lightweight `aer` tool.
-
-## Using ae
-### Commands
-
-```bash
-ae oci
-ae oci create
-ae oci delete
-ae oci kill
-ae oci start
-ae oci status
-```
-
-Implements the [OCI Command Line](https://github.com/opencontainers/runtime-tools/blob/master/docs/command-line-interface.md) interface with associated subcommands.
-
-```bash
-ae allocate
-ae allocate cell
-ae allocate pod
-```
-
-Reserve resources, and manage any prerequisites but do not start. Will fail if resources are unavailable.
-
-```bash
-ae free
-ae free cell
-ae free pod
-```
-
-Free resources, and destroy any prerequisites that have been started. Will fail if resources cannot be freed or do not exist.
-
-```bash
-ae start
-ae start executable, exe
-ae start container # Note this has an alias: 'ae oci start'
-```
-
-Run a resource immediately.
-
-```bash
-ae stop
-ae stop executable, exe
-ae stop container # Note this has an alias: 'ae oci kill'
-```
-
-Stop a resource immediately.
-
-#### Logs
-
-```bash
-ae logs <options>
-```
-
-#### Discovery
-
-```bash
-ae discover <cidr <cidr> | ip <ip>>
-```
-
-Scans a cluster of Aurae nodes and returns information about them, including the version running.
-
-#### Health
-
-```bash
-ae check <cidr <cidr> | ip <ip>> <service,...>
-```
-
-Scans a cluster of Aurae nodes and returns the serving status of the given list of services.
-
-
-## Guiding Principles
-
-### Less is more.
+### **What is `aer`?**
```

Comment:
lose the formatting inside the header please :)

Reply:
fixed in f70ca33.
excuse me for not thining to remove it in all headers. kinda stupid.

resolved.